src/components/Badge/Badge.test.tsx

```
@@ -0,0 +1,189 @@
+      expect(badge).toHaveClass('text-[#145B8F]');
+    });
+
+    it('applies resultStatus variant classes', () => {
+      render(<Badge variant="resultStatus" action="positive">Positive</Badge>);
+      const badge = screen.getByText('Positive');
+      expect(badge).toHaveClass('bg-error');
+      expect(badge).toHaveClass('text-error-800');
+    });
+
+    it('applies notification variant classes', () => {
+      render(<Badge variant="notification"/>);
```

ESLint

[error] Insert `·`

(prettier/prettier)